eg/examples/find.md

```diff
@@ -181,4 +181,8 @@ owner, group, and world (`-exec chmod a+x {} \;`):
 
     find ./searchdir -type f -name '*.php' -exec chmod a+x {} \;
 
+Correcting permissions of an unzipped file:
+
+    find ./searchdir -type d -exec chmod 755 '{}' \;
+    find ./searchdir -type f -exec chmod 644 '{}' \;
```

Comment:
I advocate using the symbolic modes (`ugoa+-=rwxXst`) instead of the octal permissions bits. Yes, the octal notation is more compact but the [symbolic mode](http://en.wikipedia.org/wiki/Chmod#Symbolic_modes) is a lot easier for people to learn and remember.

In this case the `chmod` commands would be `chmod u=rwx,go=rx '{}'`  and `chmod u=rw,go=r '{}`' respectively.